src/util/FactorUtil.js

```diff
@@ -347,12 +347,18 @@ function (Okta, TimeUtil) {
   };
 
   fn.removeRequirementsFromError = function (responseJSON, policy) {
-    var passwordRequirementsAsString = this.getPasswordComplexityDescription(policy);
+    var passwordRequirementsAsString = fn.getPasswordComplexityDescription(policy);
     if (responseJSON.errorCauses
-        && responseJSON.errorCauses.length > 0
-        && _.isString(responseJSON.errorCauses[0].errorSummary)) {
-      responseJSON.errorCauses[0].errorSummary = responseJSON.errorCauses[0].errorSummary
+        && responseJSON.errorCauses.length > 0) {
+      if (_.isString(responseJSON.errorCauses[0].errorSummary)) {
+        responseJSON.errorCauses[0].errorSummary = responseJSON.errorCauses[0].errorSummary
         .replace(passwordRequirementsAsString, '').trim();
+      }
+      // OIE type response
+      if (Array.isArray(responseJSON.errorCauses[0].errorSummary) && responseJSON.errorCauses[0].errorSummary.length) {
+        responseJSON.errorCauses[0].errorSummary = responseJSON.errorCauses[0].errorSummary[0]
+          .replace(`[${passwordRequirementsAsString}]`, '').trim();
```

Comment:
How did we end up having extra `[]`?

Reply:
There seems to be an extra square brackets around the string in `message`:
`"message":"Password requirements were not met. [Password requirements: at least 8 characters, a lowercase letter, an uppercase letter, a number, a symbol, no parts of your username, does not include your first name, does not include your last name. Your password cannot be any of your last 4 passwords. At least 2 hour(s) must have elapsed since you last changed your password.]",`